src/commands/Util/help.ts

```diff
@@ -39,11 +39,9 @@ export default class Help extends AstraeaCommand {
 			/* eslint-disable @typescript-eslint/restrict-template-expressions */
 				.setTitle(`Command | ${command.name}`)
 				.setDescription(
-					`**Description:** ${
-						command.description || '`No description`'
-					}\n**In detail:** ${
-						command.detailedDescription || '`No detailed description`'
-					}`
+					`**Aliases**: ${command.aliases.join(', ') || '`No aliases`'}
+					**Description:** ${command.description || '`No description`'}
+					**In detail:** ${command.detailedDescription || '`No detailed description`'}`
```

Comment:
If there isn't a description, you could omit that in your output to the user. I find that this practice unclutters the view, since the information that there isn't a description isn't really any help to the end user